features/find_car.feature

```diff
@@ -0,0 +1,286 @@
+Feature: Find car
+  As a user, I want to use search filters, because they give a quick and easy way to find a car that suits me.
+
+  Background:
+    Given page with filters for search cars is opened
+
+  Scenario Outline: Checking correct postcode
+    When user fill in Postcode field with <postcode>
+    And click the Search cars button
+    Then all cars possible for purchase are shown
+    And a Make filter is selected as Any
+    And a Model filter is selected as Any
+    And a Model Variant filter is selected as Any
+    And a Price filter is selected as Any
+    And an Year filter is selected as Any
+    And a Mileage filter is selected as Any
+    And a Body type filter is selected as Any
+    And a Fuel type filter is selected as Any
+    And an Engine size filter is selected as Any
+    And a Fuel consumption filter is selected as Any
+    And an Acceleration filter is selected as Any
+    And a Gearbox filter is selected as Any
+    And a Drivetrain filter is selected as Any
+    And a Doors filter is selected as Any
+    And a Seats filter is selected as Any
+    And a Insurance Group filter is selected as Any
+    And a Annual tax filter is selected as Any
+    And a Colour filter is selected as Any
+    And a Private & trade filter is selected as Any
+    And filter CO2 emissions is selected as Any too
+
+    Examples:
```

Comment:
No need to use Scenario Outline if there is only one example. It links to a lot of scenarios below.